apps/web/src/pages/tenants/components/UpdateTenantSidebar.tsx

```diff
@@ -82,61 +84,121 @@ export function UpdateTenantSidebar({
+    { identifier: string }
+  >(({ identifier }) => deleteTenant(identifier), {
+    onSuccess: async () => {
+      await queryClient.refetchQueries({
+        predicate: ({ queryKey }) => queryKey.includes(QueryKeys.tenantsList),
+      });
+      successMessage(`The tenant ${tenant?.name} is deleted`);
+    },
+    onError: (e: any) => {
+      errorMessage(e.message || 'Unexpected error');
+    },
+  });
+
+  const onDeleteTenant = async (identifier) => {
+    console.log(tenant);
+    await deleteTenantMutate({
+      identifier: identifier,
+    });
+
+    onClose();
+  };
+
   if (!tenant) {
     return null;
   }
 
   return (
-    <Sidebar
-      isOpened={isOpened}
-      onClose={onClose}
-      isLoading={isLoadingTenant}
-      onSubmit={(e) => {
-        handleSubmit(onUpdateTenant)(e);
-        e.stopPropagation();
-      }}
-      customHeader={
-        <Stack h={80}>
-          <Controller
-            control={control}
-            name="name"
-            defaultValue=""
-            render={({ field }) => {
-              return (
-                <NameInput
-                  {...field}
-                  value={field.value}
-                  data-test-id="tenant-title-name"
-                  placeholder="Enter tenant name"
-                  ml={-10}
-                />
-              );
-            }}
-          />
-        </Stack>
-      }
-      customFooter={
-        <Group position="apart" w="100%">
-          <Stack spacing={0}>
-            <Text size={'sm'} color={colors.B40}>
-              Updated at {format(new Date(tenant.updatedAt), 'dd/MM/yyyy HH:mm')}
-            </Text>
-            <Text size={'sm'} color={colors.B40}>
-              Created at {format(new Date(tenant.createdAt), 'dd/MM/yyyy HH:mm')}
-            </Text>
+    <>
+      <Sidebar
+        isOpened={isOpened}
+        onClose={onClose}
+        isLoading={isLoadingTenant}
+        onSubmit={(e) => {
+          handleSubmit(onUpdateTenant)(e);
+          e.stopPropagation();
+        }}
+        customHeader={
+          <Stack h={80}>
+            <Group position="apart" spacing={130}>
+              <Controller
+                control={control}
+                name="name"
+                defaultValue=""
+                render={({ field }) => {
+                  return (
+                    <NameInput
+                      {...field}
+                      value={field.value}
+                      data-test-id="tenant-title-name"
+                      placeholder="Enter tenant name"
+                      ml={-10}
+                    />
+                  );
+                }}
+              />
+
+              <Tooltip label={'Delete tenant'} position="bottom">
+                <div
+                  onClick={() => {
+                    setModalIsOpened(true);
+                  }}
+                >
+                  <Trash />
+                </div>
+              </Tooltip>
```

Comment:
Please wrap the icon with mantine's ActionIcon instead of the div. as it will have all the styles of a button like- pointer on hover. 
```<ActionIcon variant='transparent' onClick={...}> ```

Also, please notice that the trash icon's color should be colors.B60 and white on hover. 
You could take a look [here](https://github.com/novuhq/novu/blob/next/apps/web/src/pages/integrations/components/ConditionIconButton.tsx#L55). It should be about the same.